main.js

```diff
@@ -0,0 +1,240 @@
+window.onload = function() {
+    document.getElementById('surveyBuildingDiv').style.display = "none";
+    document.getElementById('finishButton').style.display = "none";
+    document.getElementById('finalSurveyForm').style.display = "none";    
+
+  };
+
+document.getElementById("createSurvey").onclick = function(event) {
+    document.getElementById('surveyBuildingDiv').style.display = "block";
+    document.getElementById('finishButton').style.display = "block";
+    document.getElementById('welcomeUser').style.display = "none";
+    document.getElementById('div3').style.display = "none";
+    document.getElementById('div4').style.display = "none";
+}
+
+var c = 0;
+document.getElementById("addMoreOptions").onclick = function(event) {
+    event.preventDefault(); 
+    console.log('value of c is '+ c);
+
+    c=c+1;
+    if( c == 1)
+    {
+        document.getElementById('div3').style.display = "block";
+    }
+    if( c == 2)
+    {
+        document.getElementById('div4').style.display = "block";
+        c=0;
+    }
+}
+
+
+
+document.getElementById("saveQuestion").onclick = function(e){
+    e.preventDefault();
+    addQuestion();
+
+}
+
+let survey = [];
+
+
+
+function addQuestion()
+{
+    let surveySet = {
```

Comment:
we are not preferring this, acrually you can directly create the object as {sq,options}